samples/caf_sensor_manager/configuration/nrf5340dk_nrf5340_cpuapp/event_proxy_init.c

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#define MODULE event_proxy_init
+
+#include <ipc/ipc_service.h>
+#include <event_manager_proxy.h>
+#include <caf/events/sensor_data_aggregator_event.h>
```

Comment:
this should not be needed

Reply:
It is needed as we subscribe to event: 
EVENT_MANAGER_PROXY_SUBSCRIBE(ipc_instance, sensor_data_aggregator_event);